README.md

```diff
@@ -1,126 +1,110 @@
 
-```
-NAME                                     READY   STATUS    RESTARTS   AGE
-adservice-76bdd69666-ckc5j               1/1     Running   0          2m58s
-cartservice-66d497c6b7-dp5jr             1/1     Running   0          2m59s
-checkoutservice-666c784bd6-4jd22         1/1     Running   0          3m1s
-currencyservice-5d5d496984-4jmd7         1/1     Running   0          2m59s
-emailservice-667457d9d6-75jcq            1/1     Running   0          3m2s
-frontend-6b8d69b9fb-wjqdg                1/1     Running   0          3m1s
-loadgenerator-665b5cd444-gwqdq           1/1     Running   0          3m
-paymentservice-68596d6dd6-bf6bv          1/1     Running   0          3m
-productcatalogservice-557d474574-888kr   1/1     Running   0          3m
-recommendationservice-69c56b74d4-7z8r5   1/1     Running   0          3m1s
-redis-cart-5f59546cdd-5jnqf              1/1     Running   0          2m58s
-shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
-```
+   ```sh
+   kubectl get pods
+   ```
 
-7. **Access the web frontend in a browser** using the frontend's `EXTERNAL_IP`.
+   After a few minutes, you should see the Pods in a `Running` state:
 
-```
-kubectl get service frontend-external | awk '{print $4}'
-```
+   ```
+   NAME                                     READY   STATUS    RESTARTS   AGE
+   adservice-76bdd69666-ckc5j               1/1     Running   0          2m58s
+   cartservice-66d497c6b7-dp5jr             1/1     Running   0          2m59s
+   checkoutservice-666c784bd6-4jd22         1/1     Running   0          3m1s
+   currencyservice-5d5d496984-4jmd7         1/1     Running   0          2m59s
+   emailservice-667457d9d6-75jcq            1/1     Running   0          3m2s
+   frontend-6b8d69b9fb-wjqdg                1/1     Running   0          3m1s
+   loadgenerator-665b5cd444-gwqdq           1/1     Running   0          3m
+   paymentservice-68596d6dd6-bf6bv          1/1     Running   0          3m
+   productcatalogservice-557d474574-888kr   1/1     Running   0          3m
+   recommendationservice-69c56b74d4-7z8r5   1/1     Running   0          3m1s
+   redis-cart-5f59546cdd-5jnqf              1/1     Running   0          2m58s
+   shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
+   ```
 
-*Example output - do not copy*
+7. Access the web frontend in a browser using the frontend's external IP.
 
-```
-EXTERNAL-IP
-<your-ip>
-```
+   ```
+   kubectl get service frontend-external | awk '{print $4}'
+   ```
 
-**Note**- you may see `<pending>` while GCP provisions the load balancer. If this happens, wait a few minutes and re-run the command.
+   Visit `https://EXTERNAL_IP` to access your instance of Online Boutique.
 
-8. [Optional] **Clean up**:
+8. Once you are done with it, delete the GKE cluster.
 
-```
-gcloud container clusters delete onlineboutique \
-    --project=${PROJECT_ID} --zone=${ZONE}
-```
+   ```
+   gcloud container clusters delete online-boutique
+   ```
```

Comment:
**Thought:**
For some reason, even after Online Boutique was successfully deployed, my cluster was "repairing" (for at least 10 minutes, possibly longer — I didn't wait to find out).
See my [logs here](https://screenshot.googleplex.com/44pffbbNjNyc4cu.png). See my [UI here](https://screenshot.googleplex.com/92foLdfTzECRyNo.png).
```
ERROR: (gcloud.container.clusters.delete) Some requests did not succeed:
 - ResponseError: code=400, message=Operation operation-1682103148437-d628c3f9-139c-4048-a82a-8b4749943059 is currently repairing cluster online-boutique. Please wait and try again once it is done.
```

So I couldn't delete the cluster immediately.
If you're aware that this is a recurring and common issue, feel free to add a note.
Otherwise, no action required — might be a random error.

This is really something for the autopilot PMs.

Reply:
Added a quick note